lib/entry-state.js

```diff
@@ -12,11 +13,32 @@ module.exports = async function attachEntryState (entry) {
   const publishedEntry = await getPublishedEntry(entry)
   entry.draft = false
   entry.pendingChanges = false
-
   if (!publishedEntry) {
     entry.draft = true
   }
 
+  const entriesToCompare = Object.keys(entry.fields).map((key) => {
+    const field = entry.fields[key]
+    if (isObject(field)) {
+      return [entry.fields[key], publishedEntry.fields[key]]
+    }
+  }).filter(Boolean)
+  entriesToCompare.forEach((item) => {
+    const originalItem = item[0]
+    const publishedItem = item[1]
+    if (isArray(originalItem)) {
+      originalItem.forEach((innerItem, index) => {
+        if (!isArray(innerItem) && isObject(innerItem) && (!innerItem.fields || originalItem[index].sys.updatedAt !== publishedItem[index].sys.updatedAt)) {
```

Comment:
I miss a comment here what the long check actually does and why

Reply:
👍